cmd/utils/flags.go

```diff
@@ -638,9 +638,9 @@ var (
 		Name:  "kas.sc.anchor.operator",
 		Usage: "The operator address for KAS anchor",
 	}
-	KASServiceChainXKRNFlag = cli.StringFlag{
-		Name:  "kas.xkrn",
-		Usage: "The x-krn for KAS",
+	KASServiceChainXChainIdFlag = cli.StringFlag{
+		Name:  "kas.x-chain-id",
```

Comment:
Just out of curiosity, what `x` stands for here?

Reply:
KAS team @leejh903 said it is a general prefix for header of http request. 👍 
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers

Reply:
But the `x-` style is deprecated but KAS API use the convention.